lms/djangoapps/instructor/views/api.py

```diff
@@ -3165,6 +3165,42 @@ def mark_student_can_skip_entrance_exam(request, course_id):
     return JsonResponse(response_payload)
 
 
+@method_decorator(cache_control(no_cache=True, no_store=True, must_revalidate=True), name='dispatch')
+class MarkStudentCanSkipEntranceExam(APIView):
```

Comment:
@awais786 should there be an old function that is deleted? I don't see the deletion here.

Reply:
Yes, i missed that. Thanks for catching.